### PR TITLE
Extract code about MaskType drawing into UIBezierPath extension #no-public-changes

### PR DESCRIPTION
**Pull request by phimage**
This is the first part of issue #482 
Just a refactor to move code in UIBezierPath.

This is an old closed PR
https://github.com/IBAnimatable/IBAnimatable/pull/485

(swiftlint warnings will be fixed...)

### CODE REVIEW

File: Sources/Extensions/UIBezierPathExtension.swift [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+  // MARK: - Circle
+
+  /**
+   Create a Bezier path for a circle shape.
+   
+   - Parameter bounds: The bounds of shape.
+   */
+  convenience init(circleIn bounds: CGRect) {
+    let diameter = ceil(min(bounds.width, bounds.height))
```
Comment (JakeLin) [inline Sources/Extensions/UIBezierPathExtension.swift:18]: do we need to call `self.init()` here?

**Review by JakeLin — APPROVED** [review]
👍  great stuff, good to go

**Review by tbaranes — APPROVED** [review]

File: Sources/Extensions/UIBezierPathExtension.swift (hunk 1 above)
Comment (phimage) [reply Sources/Extensions/UIBezierPathExtension.swift:18]: `self.init(` is called 3 lines under

Comment (JakeLin) [reply Sources/Extensions/UIBezierPathExtension.swift:18]: @phimage cool 👍

**Pull request merged**